django/library/jinja2/library/codebases/releases/retrieve.jinja

```diff
@@ -56,6 +56,22 @@
     ]) }}
 {% endblock %}
 
+{% block ogp_tags %}
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="{{ request.build_absolute_uri(absolute_url) }}">
+    <meta property="og:type" content="article">
+    <meta property="og:title" content="{{ codebase.title }}">
+    <meta property="og:description" content="{{ codebase.description.raw|truncate(150) }}">
```

Comment:
we should probably use `codebase.summarized_description` (and modify its implementation to make it work better for this use case as needed)